Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mkdir -p /go/src/github.com/grpc-ecosystem && \
     cp -R vendor/ /go/src/github.com/grpc-ecosystem/grpc_health_probe && \
     rm -rf /go/src/github.com/grpc-ecosystem/grpc_health_probe/vendor/github.com/grpc-ecosystem/grpc-health-probe && \
     cd /go/src/github.com/grpc-ecosystem/grpc_health_probe && \
-    CGO_ENABLED=0 go install -a -tags netgo -ldflags "-w"
+    CGO_ENABLED=0 go install -mod=mod -a -tags netgo -ldflags "-w"
```

Comment:
I think we might be able to get away with removing this from the GOPATH, which should simplify things some.